tensorboard/uploader/exporter_test.py

```diff
@@ -227,6 +229,214 @@ def outdir_files():
             },
         )
 
+    def test_e2e_success_case_with_blob_sequence_data(self):
+        """Covers exporting of finished and unfinished blob sequences
+
+        as well as rpc error during blob streaming.
+        """
+        mock_api_client = self._create_mock_api_client()
+
+        def stream_experiments(request, **kwargs):
+            del request  # unused
+            self.assertEqual(kwargs["metadata"], grpc_util.version_metadata())
+
+            response = export_service_pb2.StreamExperimentsResponse()
+            response.experiments.add(experiment_id="123")
+            yield response
+            response = export_service_pb2.StreamExperimentsResponse()
+            response.experiments.add(experiment_id="456")
+            yield response
+
+        def stream_experiment_data(request, **kwargs):
+            self.assertEqual(kwargs["metadata"], grpc_util.version_metadata())
+
+            tag = "__default_graph__"
+            for run in ("train", "test"):
+                response = export_service_pb2.StreamExperimentDataResponse()
+                response.run_name = run
+                response.tag_name = tag
+                display_name = "%s:%s" % (request.experiment_id, tag)
+                response.tag_metadata.CopyFrom(
+                    test_util.scalar_metadata(display_name)
```

Comment:
It seems a bit confusing to use the scalar plugin metadata here,
especially because it has the wrong data class. Can we at least use a
metadata value with `data_class=summary_pb2.DATA_CLASS_BLOB_SEQUENCE`?